Houdini/static/convert.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix apix:<http://fedora.info/definitions/v4/api-extension#> .
+@prefix owl:<http://www.w3.org/2002/07/owl#> .
+@prefix ebucore:<http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
+@prefix fedora:<http://fedora.info/definitions/v4/repository#> .
+@prefix islandora:<http://islandora.ca/CLAW#> .
+@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> .
+
+<> a apix:Extension;
+    rdfs:label "Image Conversion Service";
+    rdfs:comment "imagemagick's convert as a microservice";
+    apix:exposesService islandora:ConvertService;
+    apix:exposesServiceAt "svc:convert";
+    apix:bindsTo <#class> .
+
+<#class> owl:intersectionOf (
+        fedora:Binary
```

Comment:
Should we change this to `http://www.w3.org/ns/ldp#NonRDFSource` to be repository agnostic and not tie to a "Fedora" type?